src/miranda/convert/configs/nrcan-gridded-obs_cf_attrs.json

```diff
@@ -0,0 +1,95 @@
+{
+  "Header": {
+    "Conventions": "CF-1.9",
+    "_miranda_version": true,
+    "_missing_values": [
+      "-999."
+    ],
+    "_variable": true,
+    "acknowledgement": "This data is provided by Natural Resources Canada (NRCan).",
+    "author": "Natural Resources Canada",
+    "citation": "When referring to the ANUSPLIN gridded climate data set, the authors provide this data under the Environment and Climate Change Canada Data Server End Use License (<https://open.canada.ca/en/open-government-licence-canada>) and demand that any references to or derivative works based on this data set be cited as follows: McKenney, D.W., Hutchinson, M., Papadopol, P., Lawrence, K., Pedlar, J.H., Campbell, K.L., Milewska, E., Hopkinson, R.F., Price, D., Owen, T., 2011a. Customized spatial climate models for North America. Bulletin of the American Meteorological Society 92(12), pp. 1611-1622. https://doi.org/10.1175/2011BAMS3132.1",
+    "contact": "dan.mckenney@NRCan-RNCan.gc.ca",
+    "documentation": "https://www.canada.ca/en/environment-climate-change/services/climate-change/canadian-centre-climate-services/display-download/technical-documentation-adjusted-climate-data.html",
+    "domain": "CAN",
+    "frequency": "day",
+    "institution": "GovCan",
+    "license": "https://open.canada.ca/en/open-government-licence-canada",
+    "license_type": "permissive",
+    "organization": "NRCan",
+    "processing_level": "raw",
+    "product": "Natural Resources Canada ANUSPLIN interpolated historical climate model dataset",
+    "realm": "atmos",
```

Comment:
is this a standard ?

Reply:
No. It shouldn't be there either, IMO. Good call.